docs/resources.js

```diff
@@ -51,25 +50,30 @@ class DisasterMapValue {
 
   /** @return {string} The combined SNAP/damage asset name */
   getCombinedAsset() {
-    return 'users/gd/' + this.name + '/data-ms-as-nod';
+    return gdEePathPrefix + this.name + '/data-ms-as-nod';
   }
 }
 
-// TODO: Don't store census/svi data in relation to a disaster so we can handle
-// scenarios that are not 1:1 state:disaster e.g. michael which hit both
-// florida and georgia.
+// TODO(janakr): get all the data below here from Firestore instead of hard-
+//  coding here. That will also need to support multiple states per disaster.
+const michaelName = '2018-michael';
+const michaelPathPrefix = gdEePathPrefix + michaelName + '/';
+
+const harveyName = '2017-harvey';
+const harveyPathPrefix = gdEePathPrefix + harveyName + '/';
+
 disasters.set(
-    'michael',
+    michaelName,
     new DisasterMapValue(
-        'michael', 2018, 'users/gd/michael/FEMA_Damage_Assessments',
-        'users/gd/michael/snap', 'users/gd/michael/tiger',
-        'users/gd/michael/income', 'users/gd/michael/svi',
-        'users/gd/michael/relevant_buildings'));
+        michaelName, michaelPathPrefix + 'FEMA_Damage_Assessments',
```

Comment:
Maybe add a TODO to not store the path prefix for each of these since we can infer it form the disaster name?

Reply:
I think that we can't in all cases (because we might be reading directly from the state table, right?). But in any case, this will be mostly made moot by the previous TODO, that all this data will be read in from Firestore. If we want to store the data in Firestore without prefixes, I guess we can, but not sure it's worth it for the potential confusion.